### PR TITLE
Add action recommendation to 404 error.

### PR DESCRIPTION
**Pull request by rmweir**
Problem: CLI returns 404 when issuing "rancher project ls" command after toggle rancher project "add-local" argument from true to false. This will actually occur whenever newClient() is called by any method if project is invalid.

Solution: Added message to error produced by go routines cliclient.newMasterClient(). 

Issue: rancher/rancher#16981

### CODE REVIEW

**Comment by dramich** [comment]
LGTM

**Comment by cjellick** [comment]
The subject line of your commit looks too long @rmweir 
see https://chris.beams.io/posts/git-commit/

**Comment by cjellick** [comment]
LGTM

File: cliclient/cliclient.go [hunk 1]
```diff
@@ -114,6 +115,9 @@ func (mc *MasterClient) newClusterClient() error {
 	// Setup the project client
 	cc, err := clusterClient.NewClient(options)
 	if err != nil {
+		if apiError, ok := err.(*clientbase.APIError); ok && apiError.StatusCode == 404 {
```
Comment (cjellick) [inline cliclient/cliclient.go]: @dramich dont we have access to a `IsNotFound` helper function

**Comment by cjellick** [comment]
LGTM

**Pull request merged**